crates/voicevox_core_python_api/python/test/test_audio_query.py

```diff
@@ -23,16 +25,78 @@ def test_accept_json_without_optional_fields() -> None:
                       "accent": 1
                     }
                   ],
-                  "speed_scale": 1.0,
-                  "pitch_scale": 0.0,
-                  "intonation_scale": 1.0,
-                  "volume_scale": 1.0,
-                  "pre_phoneme_length": 0.1,
-                  "post_phoneme_length": 0.1,
-                  "output_sampling_rate": 24000,
-                  "output_stereo": false
+                  "speedScale": 1.0,
+                  "pitchScale": 0.0,
+                  "intonationScale": 1.0,
+                  "volumeScale": 1.0,
+                  "prePhonemeLength": 0.1,
+                  "postPhonemeLength": 0.1,
+                  "outputSamplingRate": 24000,
+                  "outputStereo": false
                 }
                 """,
             )
         )
     )
+
+
+def test_dumps() -> None:
+    BEFORE = textwrap.dedent(
+        """\
+        {
+          "accent_phrases": [],
+          "speedScale": 1.0,
+          "pitchScale": 0.0,
+          "intonationScale": 1.0,
+          "volumeScale": 1.0,
+          "prePhonemeLength": 0.1,
+          "postPhonemeLength": 0.1,
+          "outputSamplingRate": 24000,
+          "outputStereo": false,
+          "pauseLength": null,
+          "pauseLengthScale": 1.0,
+          "kana": ""
+        }""",
+    )
+
+    after = json.dumps(dataclasses.asdict(AudioQuery(**json.loads(BEFORE))), indent=2)
+    assert BEFORE == after
+
+
+# あまり保証したくない性質ではあるが、`dataclasses.asdict`に必要
+def test_getattr() -> None:
+    query = AudioQuery(
+        **json.loads(
+            textwrap.dedent(
+                """\
+                {
+                  "accent_phrases": [],
+                  "speedScale": 1.0,
+                  "pitchScale": 0.0,
+                  "intonationScale": 1.0,
+                  "volumeScale": 1.0,
+                  "prePhonemeLength": 0.1,
+                  "postPhonemeLength": 0.1,
+                  "outputSamplingRate": 24000,
+                  "outputStereo": false
+                }
+                """,
+            )
+        )
+    )
+
+    assert getattr(query, "speedScale") is query.speed_scale
```

Comment:
`accent_phrases`もassert含めたほうが良いかも？
これだけはsnake_caseになるというテストとして。

```suggestion
    assert getattr(query, "accent_phrases") is query.speed_scale
```

Reply:
[`09ce920` (#946)](https://github.com/VOICEVOX/voicevox_core/pull/946/commits/09ce9206231b9fcd61223a647962ee3858041262): `kana`も含めました。